frontend/src/components/CounterfactualStatistics.jsx

```diff
@@ -5,11 +5,15 @@ import PropTypes from "prop-types";
 const propTypes = exact({
   totalCasesCounterfactual: PropTypes.number,
   totalCasesReal: PropTypes.number,
+  shiftFirstRestrictions: PropTypes.instanceOf(Date).isRequired,
+  shiftLockdown: PropTypes.instanceOf(Date).isRequired,
 });
 
 const defaultProps = {};
 
 const CounterfactualStatistics = (props) => {
+  // divide to turn shift from date difference to days
+  const to_days = 1000 * 3600 * 24;
```

Comment:
Maybe this is better as a function? Something like:

```
function daysFromDate(inputDate) {
  return (inputDate / (1000 * 3600 * 24));
}
```

Also, can you explain why dividing converts a date into a number of days? I'm not sure I get that bit.

Reply:
Done. The date difference returns it in milliseconds, that is why we need to divide it.